### PR TITLE
Sitton release cargo

### PR DESCRIPTION
**Pull request by tal-sitton**

### CODE REVIEW

File: src/main/java/frc/robot/Commands/IntakeCargo.java [hunk 1]
```diff
@@ -26,7 +26,7 @@ public boolean isFinished() {
                                         return intakeToTransfer.getLimit();
                                     }
                                 },
-                                new MoveGenericSubsystem(transfer, Transfer.SPEED) {
+                                new MoveGenericSubsystem(transfer, Transfer.getInstance().getTransferSpeed()) {
```
Comment (OfriRosenbaum) [inline src/main/java/frc/robot/Commands/IntakeCargo.java:29]: you already have the instance, why would you use `getInstance()` again and not the instance you already have?

File: src/main/java/frc/robot/Commands/IntakeCargo.java [hunk 2]
```diff
@@ -40,7 +40,7 @@ public boolean isFinished() {
                             }
                         }
                 ),
-                new MoveGenericSubsystem(transfer, Transfer.SPEED) {
+                new MoveGenericSubsystem(transfer, Transfer.getInstance().getTransferSpeed()) {
```
Comment (OfriRosenbaum) [inline src/main/java/frc/robot/Commands/IntakeCargo.java:43]: ditto

**Review by OfriRosenbaum — DISMISSED** [review]

**Comment by OfriRosenbaum** [comment]
Also, what is this branch name?

**Comment by tal-sitton** [comment]
> Also, what is this branch name?

its intellij's default,  for some reason...

**Review by yutytuty — APPROVED** [review]

**Pull request merged**